vassal-doc/src/main/readme-referencemanual/ReferenceManual/Properties.adoc

```diff
@@ -57,35 +57,35 @@ the level in a <<Layer.adoc#Properties,Layer)>> you can use _<_, _<=_, _>_, and
 ==== Handy Combined List of VASSAL-defined Properties for Game Pieces
 
 |===
-|Property |Trait |Description
+|*Property* |*Trait* |*Description*
 |*BasicName* |<<BasicPiece.adoc#top,Basic Piece>> |The basic name of the piece.
-|*ClickedX* |Basic Piece |Map X-coordinate where player last right-clicked on piece to bring up context menu (or 0 if never).
-|*ClickedY* |Basic Piece |Map Y-coordinate where player last right-clicked on piece to bring up context menu (or 0 if never).
-|*CurrentBoard* |Basic Piece |Current Board name or "" if not on a map.
-|*CurrentMap* |Basic Piece |Current Map name or "" if not on a map.
-|*CurrentX* |Basic Piece |Current X-coordinate of the piece.
-|*CurrentY* |Basic Piece |Current Y-coordinate of the piece.
-|*CurrentZone* |Basic Piece |Current Zone name if the current map has a <<ZonedGrid.adoc#top,multi-zoned grid>>, or "" if the piece is not in any zone, or not on a map.
+|*ClickedX* |<<BasicPiece#top,Basic Piece>>|Map X-coordinate where player last right-clicked on piece to bring up context menu (or 0 if never).
+|*ClickedY* |<<BasicPiece#top,Basic Piece>>|Map Y-coordinate where player last right-clicked on piece to bring up context menu (or 0 if never).
+|*CurrentBoard* |<<BasicPiece#top,Basic Piece>>|Current Board name or "" if not on a map.
+|*CurrentMap* |<<BasicPiece#top,Basic Piece>>|Current Map name or "" if not on a map.
+|*CurrentX* |<<BasicPiece#top,Basic Piece>>|Current X-coordinate of the piece.
+|*CurrentY* |<<BasicPiece#top,Basic Piece>>|Current Y-coordinate of the piece.
+|*CurrentZone* |<<BasicPiece#top,Basic Piece>>|Current Zone name if the current map has a <<ZonedGrid.adoc#top,multi-zoned grid>>, or "" if the piece is not in any zone, or not on a map.
 |*DeckName* |Basic Piece |Current Deck name if stacked in a <<Deck.adoc#top,Deck>>, or "" if not in a Deck.
```

Comment:
Did you deliberately not add the link to the BasicPiece document to DeckName, or was that an oversight?